ui/imports/shared/status/StatusStickerPackClickPopup.qml

```diff
@@ -80,7 +82,7 @@ ModalPopup {
                 preSelectedRecipient: stickerPackDetailsPopup.store.stickersStore.getStickersMarketAddress()
                 preDefinedAmountToSend: LocaleUtils.numberToLocaleString(parseFloat(price))
                 preSelectedHoldingID: {
-                    let token = ModelUtils.getByKey(root.walletAssetsStore.groupedAccountAssetsModel, "tokensKey", stickerPackDetailsPopup.store.stickersStore.getStatusTokenKey())
+                    let token = SQUtils.ModelUtils.getByKey(stickerPackDetailsPopup.walletAssetsStore.groupedAccountAssetsModel, "tokensKey", stickerPackDetailsPopup.store.stickersStore.getStatusTokenKey())
```

Comment:
Buying stickers is still somewhat broken, this just launches the SendModal but spits out a lot of warnings and errors; not sure we have an existing issue for this? CC @jrainville @micieslak

Reply:
Were you testing on testnet or mainnet? I'm not sure if we support testnet for stickers (since people could just buy them on testnet and not pay real money)

Reply:
On main net